src/Ticket.java

```diff
@@ -83,6 +83,11 @@ public void setStadiumSector(char sector) {
         this.stadiumSector = sector;
     }
 
+    // added by me
```

Comment:
It would be better to explain why it was added (// added by me seems unclear).

Reply:
In general, getters and setters are just standard methods without any description.))